docs/style.css

```diff
@@ -36,12 +50,33 @@
   color: var(--theme-foreground-faint);
 }
 
-.observablehq-link-active a[target=_blank]:not(:hover, :focus)::after {
+.observablehq-link-active a[target="_blank"]:not(:hover, :focus)::after {
   color: var(--theme-foreground-focus);
 }
 
 #observablehq-main {
   min-height: calc(100vh - 23.8rem);
+  font-size: 16px;
+  font-weight: 500;
+}
+
+p a[href] {
+  color: inherit;
+  text-decoration: underline;
+}
```

Comment:
This should probably be applied to `a[href]` , not `p a[href]`. There’s no guarantee that links only appear within paragraph elements.

Reply:
makes sense, but then then it should be 

```
#observablehq-main a[href]:not(.observablehq-header-anchor)
```

we don't want to underline all the anchors in the copy text, nor the links in the site chrome (sidebar, toc…)